edgelet/iotedge/src/main.rs

```diff
@@ -50,6 +50,11 @@ fn run() -> Result<(), Error> {
         edgelet_core::version().replace("~", "-")
     );
 
+    let default_support_bundle_name = format!(
+        "support_bundle{}.zip",
+        chrono::Utc::now().format("_%Y_%m_%d_%H_%M_%S_%Z")
```

Comment:
No strong opinions, but how did you settle on this format? Relevant link recommends ISO 8601 format (2013-04-01T13:01:02):
https://softwareengineering.stackexchange.com/questions/61683/standard-format-for-using-a-timestamp-as-part-of-a-filename

Reply:
No real criteria - This is just a favorite format of mine that I've used previously. Wanted to avoid hyphens and colons ( as they can be challenging to deal with in filenames) and went with underscores in place of them.  Also, I wanted to get the time zone info in the filename because the devices could be in various geographies. The biggest quality I was aiming for was readability.